.github/workflows/ogf_workflow.yml

```diff
@@ -0,0 +1,395 @@
+              else
+                  echo "$functionsummaryfile failed  | Generating a new badge https://img.shields.io/badge/$badge_id-$HOST_VERSION-$FAIL_COLOR.svg and will save to $BADGE_FOLDER_NAME/$functionsummaryfile.svg"
+                  count+=1
+                  badge_filename=$(generate_badge $badge_id $HOST_VERSION $FAIL_COLOR $functionsummaryfile $BADGE_FOLDER_NAME)
+              fi
+
+              upload_badge $badge_filename $blob_name $BADGES_STORAGE_ACCOUNT_NAME $BADGES_SCENARIO_STORAGE_CONTAINER_NAME
+            done
+
+          echo "count: $count"
+          local overall="Overall"
+          local badge_filename=""
+          local badge_id="$overall"
+          local blob_name="$SKU$UNDER$REGION$UNDER$overall"
+
+          if [ $count -eq "0" ]; then
+            # SKU Region Region Host_version
+            echo "$overall passed | Generating a new badge https://img.shields.io/badge/$badge_id-$HOST_VERSION-$PASS_COLOR.svg and will save to $BADGE_FOLDER_NAME/$overall.svg"
+            badge_filename=$(generate_badge $badge_id $HOST_VERSION $PASS_COLOR $overall $BADGE_FOLDER_NAME)
+          elif [ $count -lt 4 ]; then
+            echo "$overall partially failed | Generating a new badge https://img.shields.io/badge/$badge_id-$HOST_VERSION-$PARTIAL_PASS_COLOR.svg and will save to $BADGE_FOLDER_NAME/$overall.svg"
+            badge_filename=$(generate_badge $badge_id $HOST_VERSION $PARTIAL_PASS_COLOR $overall $BADGE_FOLDER_NAME)
+          else
+            echo "$overall failed | Generating a new badge https://img.shields.io/badge/$badge_id-$HOST_VERSION-$FAIL_COLOR.svg and will save to $BADGE_FOLDER_NAME/$overall.svg"
+            badge_filename=$(generate_badge $badge_id $HOST_VERSION $FAIL_COLOR $overall $BADGE_FOLDER_NAME)
+          fi
+          
+          upload_badge $badge_filename $blob_name $BADGES_STORAGE_ACCOUNT_NAME $BADGES_OVERALL_STORAGE_CONTAINER_NAME
+          
+          local test_date=$(date -u | sed 's/ /\%20/g')  # url encoding the date for imgsheilds
+          local message="TestTimeUTC"
+          local blob_name="$SKU$UNDER$REGION$UNDER$message"
+          local badge_id="$message"
+          badge_filename=$(generate_badge $badge_id $test_date black $overall $BADGE_FOLDER_NAME)
+
+          upload_badge $badge_filename $blob_name $BADGES_STORAGE_ACCOUNT_NAME $BADGES_TIMEUTC_STORAGE_CONTAINER_NAME
+
+          local test_date=$(date -u | sed 's/ /\%20/g')  # url encoding the date for imgsheilds
+          local message="LastSuccessfulRunNumber"
+          local blob_name="$SKU$UNDER$REGION$UNDER$message"
+          local badge_id="$message"
+          badge_filename=$(generate_badge $badge_id ${{ github.run_number }} white $overall $BADGE_FOLDER_NAME)
+
+          upload_badge $badge_filename $blob_name $BADGES_STORAGE_ACCOUNT_NAME $BADGES_TIMEUTC_STORAGE_CONTAINER_NAME
+          }
+
+          process_results
+
+      # - name: Upload Logs and Records
```

Comment:
Since we're using actions/upload-artifact, I don't think we need the azure blob storage anymore.

Reply:
Yes - I commented this section out this later. Will remove this section.